recipes/pyaerocom/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "pyaerocom" %}
+{% set version = "0.8.0rc1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8fcf2cf511cc11b3ff4bee81d742a0c8d67c8e5617590be844fbfbe769193a05
+
+build:
+  number: 4
```

This should be zero, the number only needs to be bumped for builds that are deployed to anaconda.org.
```suggestion
  number: 0
```